src/test/java/org/opentripplanner/routing/algorithm/raptor/transit/request/RoutingRequestTransitDataProviderFilterTest.java

```diff
@@ -147,4 +154,49 @@ private TripTimes createTestTripTimes() {
 
     return new TripTimes(trip, List.of(stopTime), new Deduplicator());
   }
+  @Test
+  public void includePlannedCancellationsTest() {
+    TripTimes tripTimes1 = createTestTripTimes();
+    TripTimes tripTimes2 = createTestTripTimes();
```

Comment:
If these were named:

tripTimesWithCancellation
tripTimesWithReplaced

The reading of the rest of the test would be easier.